node/network/collator-protocol/src/validator_side/mod.rs

```diff
@@ -1164,6 +1212,15 @@ where
 			res = state.collation_fetches.select_next_some() => {
 				handle_collation_fetched_result(&mut ctx, &mut state, res).await;
 			}
+			res = state.collation_fetch_timeouts.select_next_some() => {
+				let (collator_id, relay_parent) = res;
```

Comment:
```suggestion
			(collator_id, relay_parent) = state.collation_fetch_timeouts.select_next_some() => {
```

Reply:
That did not compile (select macro parser?).